app/services/itsi_authoring/editor.rb

```diff
@@ -33,22 +33,21 @@ def section_json(page)
     }
   end
 
-  def embeddable_json(embeddable)
-    type = embeddable.class.name.underscore.gsub('embeddable/', '') # e.g. embeddable/open_response -> open_response
-    result = case type
-               when 'open_response'
-                 open_response_json(embeddable)
-               when 'image_question'
-                 image_question_json(embeddable)
-               else
-                 {}
-             end
-    result[:type] = type
-    result
+  def embeddable_json(e)
+    type = e.class.name.underscore.gsub('embeddable/', '') # e.g. embeddable/open_response -> open_response
+    case type
+      when 'open_response'
+        open_response_json(e)
+      when 'image_question'
+        image_question_json(e)
+      else
+        {}
+    end
   end
 
   def open_response_json(e)
     {
+      type: 'open_response',
       name: e.name,
       prompt: e.prompt,
       is_hidden: e.is_hidden,
```

Comment:
Could you add the default_text here too?  Sorry, I forgot to mention that in the email.